test/kubemark/start-kubemark.sh

```diff
@@ -338,8 +353,11 @@ function wait-for-hollow-nodes-to-run-or-timeout {
 }
 
 ############################### Main Function ########################################
+detect-project &> /dev/null
```

Comment:
Could add this as a skeleton function.  An external provider would use ```detect-project``` to search for an existing Kubernetes master, but that can also be handled in another patch.